google-cloud-bigquery/pom.xml

```diff
@@ -58,9 +58,12 @@
     <dependency>
       <groupId>org.easymock</groupId>
       <artifactId>easymock</artifactId>
-      <version>3.4</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.objenesis</groupId>
+      <artifactId>objenesis</artifactId>
+    </dependency>
```

Comment:
Its scope is defined to be test in dependency management section in the parent pom.